index.html

```diff
@@ -1,12 +1,149 @@
 	<title>Bitcoin-Conference 2021 </title>
 </head>
+
 <body>
-	
+	<!-- Header section of the mobile website version starts here -->
+	<header>
+		<nav>
+			<button type="button" class="btn hambuger-menu"><svg xmlns="http://www.w3.org/2000/svg" width="30"
+					height="30" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
+					<path stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="M4 6h16M4 12h16M4 18h16" />
+				</svg></button>
+		</nav>
+		<section class="main-wrapper">
+			<div class="main-nav">
+				<ul id="main-bar">
+					<li><i class="fab fa-facebook-f"></i></li>
+					<li><i class="fab fa-twitter"></i></li>
+					<li><a href="#">Logout</a></li>
+					<li><a href="#">Mypage</a></li>
+					<li><a href="#">English</a></li>
+				</ul>
+			</div>
+			<div class="program-section">
+				<button class="nav__close" id="close-button" type="button">
+					<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor"
+						viewBox="0 0 24 24" stroke="currentColor">
+						<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+							d="M6 18L18 6M6 6l12 12" />
+					</svg>
+				</button>
+				<a href="index.html"> <img src="Assets/Icons/running.jpeg" id="logo" alt="logo"></a>
+				<ul class="nav-menu">
+					<li class="about"><a href="about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsors</a></li>
+					<li><a href="#">News</a></li>
+					<li id="camp"><a href="#">Campaign</a></li>
+				</ul>
+			</div>
+		</section>
+		<section class="home-page">
+			<p class="moto">Keeping You Healthy</p>
+			<h1 class="main-header">
+				RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT <br /> 2021 </h1>
+			<p class="main-text">Aqua's Gym is the Number one training facility in the west-side, A joyfull arena
+				where
+				we make you meet your fitness Goals happening yearly in over 30 states.</p>
+			<time datetime="2021-08-20"> 2021.08.20(FRI) &sim; 26(THU) </time>
+			<blockquote>&commat; Mana Convention Center Wynwood Miami</blockquote>
+		</section>
+	</header>
+	<section class="main-program">
+		<h2>Main Program</h2>
+		<div class="event-container">
+			<article class="main-event">
+				<img src="Assets/Icons/lecture.png" alt="lecture-flaticon">
+				<h3>Lecture</h3>
+				<p class="lectures">Listen to top fitness coaches and experts as they share their knowledge on how to
+					live a healthy
+					lifestle.</p>
+			</article>
+			<article class="main-event">
+				<img src="Assets/Icons/exhibiiton.png" alt="exhibition-flaticon">
+				<h3>Exhibition</h3>
+				<p class="workshop">Take a look around and get inspired by the different ways you can keep yourself fit
+					and healthy.
+				</p>
+			</article>
+			<article class="main-event">
+				<img src="Assets/Icons/forum.png" alt="forum-flaticon">
+				<h3>Forum</h3>
+				<p class="forum">Have the opportunity to share your thoughts and opinions with experts on topics that
```

Comment:
- [ ] As requested by the previous Code Reviewer, kindly reduce the spacings between each article in order to be in sync and consistent with the provided design template. Below is a screenshot of the project requirement:


![image](https://user-images.githubusercontent.com/16058470/125108017-601d7380-e0d9-11eb-976c-8e09f7f5da1b.png)


Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/125107403-b3db8d00-e0d8-11eb-9c25-67254e265838.png) | ![image](https://user-images.githubusercontent.com/16058470/125107489-cc4ba780-e0d8-11eb-904b-4ecaf48ac36b.png)

Reply:
> * [x]  As requested by the previous Code Reviewer, kindly reduce the spacings between each article in order to be in sync and consistent with the provided design template. Below is a screenshot of the project requirement:
> 
> ![image](https://user-images.githubusercontent.com/16058470/125108017-601d7380-e0d9-11eb-976c-8e09f7f5da1b.png)
> 
> Yours	Original
> ![image](https://user-images.githubusercontent.com/16058470/125107403-b3db8d00-e0d8-11eb-9c25-67254e265838.png)	![image](https://user-images.githubusercontent.com/16058470/125107489-cc4ba780-e0d8-11eb-904b-4ecaf48ac36b.png)

I'm not sure you followed this thread at all mate, you just saw my ticket and went ahead asking me to make changes,  that was my last review because you folks are incompetent to do the jobs you are being paid to do. I have tried my best to keep my cool but I just lost it now. did you even read the last changes asked by the last Tse or what.  no man that's it, you all incompetence have succeded in making me give up on this all shit.

Reply:
> > * [x]   As requested by the previous Code Reviewer, kindly reduce the spacings between each article in order to be in sync and consistent with the provided design template. Below is a screenshot of the project requirement:
> > 
> > ![image](https://user-images.githubusercontent.com/16058470/125108017-601d7380-e0d9-11eb-976c-8e09f7f5da1b.png)
> > Yours	Original
> > ![image](https://user-images.githubusercontent.com/16058470/125107403-b3db8d00-e0d8-11eb-9c25-67254e265838.png)	![image](https://user-images.githubusercontent.com/16058470/125107489-cc4ba780-e0d8-11eb-904b-4ecaf48ac36b.png)
> 
> I'm not sure you followed this thread at all mate, you just saw my ticket and went ahead asking me to make changes, that was my last review because you folks are incompetent to do the jobs you are being paid to do. I have tried my best to keep my cool but I just lost it now. did you even read the last changes asked by the last Tse or what? no man that's it, you all incompetence have succeded in making me give up on this all shit.

does squashing this article that closes look really well from your point of view as a designer?  where exactly is my own inputs working on this project, even when I do believe I should follow the design when I notice something won't work isn't that where my own knowledge should come in? or what exactly is the purpose of this project? because I believe some of you are putting your own personal preferences while neglecting the sole purpose of this project, hence which is affecting my productivity and mental health. i didn't sign up for any of the latter.